docs/proposals/ListView.md

```diff
@@ -0,0 +1,153 @@
+# Component Name
+
+The `ListPage` component will provide the structure and parts needed to display
+a list view.
+
+## Design Patterns
+
+There is a desire to provide developers with page templates to both speed up the
+development of pages but also provide our users with a more consistent
+experience.
+
+A first step in this is imagining what a common `ListPage` component would look
+like. It would need to have some level of flexibility to allow for use on
+different pages but still provide common "slots" or areas where.
+
+## Accessibility
+
+As a primarily layout component we don't have a lot of interaction concerns on
+the ListPage but, we should ensure that we're using semantically appropriate
+tags wherever possible to aid in readability and navigation.
+
+## Responsiveness & Mobile
+
```

Comment:
What is taking care of the page title and the action buttons? How do they relate to this component, aren't they on the page?

Reply:
Looked at this in the interface section.